core/domain/opportunity_services.py

```diff
@@ -654,8 +675,17 @@ def increment_question_counts(skill_ids, delta):
         delta: int. The delta for which to increment each question_count.
     """
     updated_skill_opportunities = (
-        _get_skill_opportunity_with_updated_question_count(skill_ids, delta))
-    _save_skill_opportunities(updated_skill_opportunities)
+        _get_skill_opportunities_with_updated_question_counts(skill_ids, delta))
+    to_be_updated_opportunities = []
```

Comment:
Maybe `opportunities_to_delete` and `opportunities_to_update`? (I'm fine with either!)